### PR TITLE
Create PR in operator repo on javaagent release.

### PR DESCRIPTION
**Pull request by anuraaga**
@pavolloffay Can you approve or reject this PR?

### CODE REVIEW

**Review by trask — APPROVED** [review]
this is a good idea 👍

will wait for @pavolloffay before merging

**Comment by trask** [comment]
> this is a good idea 👍
> 
> will wait for @pavolloffay before merging

we could probably even send a PR using @opentelemetry-java-bot which has signed the CLA if it's just a simple version bump over there

**Comment by anuraaga** [comment]
Yikes @trask I accidentally force pushed your change, can you push it back from local?

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -150,10 +150,54 @@ jobs:
...
+          msg="Update opentelemetry-javaagent version to $VERSION"
+          git commit -a -m "$msg"
+
+          # gh pr create doesn't have a way to explicitly specify different head and base
+          # repositories currently, but it will implicitly pick up the head from a different
+          # repository if you set up a tracking branch
+
+          git push --set-upstream origin update-opentelemetry-javaagent-to-$VERSION
+
+          gh pr create --title "$msg" \
```
Comment (pavolloffay) [inline .github/workflows/release.yml:193]: It seems like the action creates a PR and it does not open an issue as the title of this PR says.

**Comment by pavolloffay** [comment]
@trask the ideal solution would if the GHA could open a PR and update https://github.com/open-telemetry/opentelemetry-operator/blob/main/autoinstrumentation/java/version.txt#L1 to match the released version. 

Opening an issue is good as well.

**Comment by anuraaga** [comment]
Thanks @pavolloffay - fixed the PR title

**Review by mateuszrzeszutek — APPROVED** [review]

**Review by pavolloffay — APPROVED** [review]

**Pull request merged**